.github/workflows/CI-linter.yml

```diff
@@ -131,5 +98,5 @@ jobs:
           LOG_FILE: superlinter.log
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # exclude link to markdown files
+          # exclude markdown file with links
           FILTER_REGEX_EXCLUDE: "docs/source/docker_files.md"
```

Comment:
what is actually the problem with the links?

Reply:
There is a markdown rule which requires that the first line is a header line (‘# This is a Header’). This is why the linter would fail.

Reply:
But we actually don't need this anymore; we have moved markdownlint to pre-commit and disabled this check (`"--disable", "MD041"]` see PR 940. 

I remove this line now.